tools/mo/automation/package_BOM.txt

```diff
@@ -41,12 +41,10 @@ openvino/tools/mo/back/MatMulNormalizer.py
 openvino/tools/mo/back/MaxPool.py
```

Comment:
can we have package BOM testing for new `convert_model`? Will it be helpful?

Reply:
do we need BOM at all?

Reply:
It checks a license at least

Reply:
Done.